pint_pandas/testsuite/test_pandas_extensiontests.py

```diff
@@ -456,7 +537,10 @@ def test_reduce_scaling(self, data, all_numeric_reductions, skipna):
         # min/max with empty produce numpy warnings
         with warnings.catch_warnings():
             warnings.simplefilter("ignore", RuntimeWarning)
+            # try:
```

Comment:
?

Reply:
I put a try statement there to catch attribute errors (in lines 542-543), but commented it out when I resolved the problems elsewhere.  I've deleted both the commented `try` and the commented `except` clauses below.